examples/nuxt/.runTest.ts

```diff
@@ -0,0 +1,18 @@
+import { page, urlBase, autoRetry, run } from '../../libframe/test/setup'
+
+export { runTest }
+
+function runTest(cmd: 'npm run dev' | 'npm run prod') {
+  run(cmd)
+
+  test('telefunction call', async () => {
+    await page.goto(`${urlBase}/`)
+    // `autoRetry` to ensure JavaScript has loaded & executed
+    await autoRetry(async () => {
```

Comment:
I'd suggest to:

```js
    await autoRetry(async () => {
      const text = await page.textContent('#view')
      expect(text).toContain('First name: Alan')
    })
    expect(text).toContain('Last name: Turing')
    expect(text).toContain('server: true')
```

Makes the intention clearer thus the code easier to read.